### DISCUSSION TITLE
Allow to set a precise temperature

### DISCUSSION THREAD

**Discussion by plunkets** [ncosgray/cuppa_mobile#124] [Ideas]
Hi,

I like Cuppa and use it for my tea and my coffee. But since i chose temps generally between 86°C and 89°C, i write them in the description field. It would be great to have the option to define a more precise temp, combined with the current +5/-5 selector. Maybe with adding two options, liké:
 -5 -1 <temp> +1 +5 

Thanks. Have a nice day everyone!

**Comment by ncosgray** [comment]
I agree -- Cuppa needs more granular temperatures. I'm working on that!

In the meantime, I've created issue #138 from this discussion.

**Comment by ncosgray** [comment]
Thanks for the excellent suggestion! With today's [Cuppa 2.6.0](https://github.com/ncosgray/cuppa_mobile/releases/tag/2.6.0) release, you'll be able to set precise brewing temperatures using the slider. For those wishing to keep using 5 degree C increments, the up/down buttons still use those temperature intervals. I think it works well and I hope you agree too.
